uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -488,11 +481,16 @@ define([
     function setCalendarProps(contactID, calendar) {
       var dateObj,
         leaveRequest,
-        dates = Object.keys(calendar.days);
+        dates = Object.keys(calendar.days),
+        indexedByMonth = angular.copy(vm.months.map(function () {
+          return [];
+        }));
 
       dates.forEach(function (date) {
         dateObj = calendar.days[date];
-        leaveRequest = getLeaveRequestByDate(contactID, dateObj.date);
+        indexedByMonth[parseInt(moment(dateObj.date).month())].push(dateObj);
```

Comment:
momentjs `month()` already returns a number (https://momentjs.com/docs/#/get-set/month/), is the `parseInt` needed?